tests/zfs-tests/tests/functional/hkdf/run_hkdf_test.ksh

```diff
@@ -0,0 +1,30 @@
+#!/bin/ksh
+
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2017 by Datto Inc. All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+
+#
+# DESCRIPTION:
+#	Call the hkdf_test tool to test ZFS's HKDF implementation against
+#	a few test vectors.
+#
+
+log_assert "Run the tests for the HKDF algorithm."
+
+log_must $STF_SUITE/tests/functional/hkdf/hkdf_test
```

Comment:
If you add `hkdf_test` to `ZFSTEST_FILES` in `tests/zfs-tests/include/commands.cfg` a link to it will be created in the constrained path and you can rewrite this as:

```
log_must hkdf_test
```